common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -2092,116 +2092,6 @@ create_religious_head_effect = {
 	}
 }
 
-plant_world_tree_effect = {
```

Comment:
You removed this effect, but it's still used in events.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/70820595-7c203c80-1df2-11ea-8166-da02ac863b8e.png)

</details>